modules/odc-deploying-java-applications.adoc

```diff
@@ -2,40 +2,42 @@
 [id="odc-deploying-java-applications_{context}"]
 = Upload JAR files for easy deployment of Java applications
 
-You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options:
+You can use the web console *Developer* perspective to upload a JAR file by using the following options:
 
 * Navigate to the *+Add* view of the *Developer* perspective, and click *Upload JAR file* in the *From Local Machine* tile. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
```

Comment:
```suggestion
* Navigate to the *+Add* view of the *Developer* perspective, and click *Upload JAR file* in the *From Local Machine* tile. Browse and select your JAR file, or drag a JAR file to deploy your application.
```

From the IMB style guideline:
*drag and drop (verb)*: Do not use. Use "drag", which includes the action of dropping an object.